pyfstat/make_sfts.py

```diff
@@ -97,9 +97,10 @@ def __init__(
             frequency evolution and amplitude parameters for injection
             if h0==None or h0==0, these are all ignored
             if h0>0, then Alpha, Delta, cosi need to be set explicitly
-        Tsft: float
-            the sft duration
-        noiseSFTs: str
+        Tsft: int
+            The SFT duration in seconds.
+            Will be ignored if noiseSFTs are given.
```

Comment:
But a warning will be thrown if different than the one in the SFTs...?

Reply:
Yes, or an error if inconsistent between IFOs.